ledger-service/http-json/src/it/scala/http/HttpServiceIntegrationTestUserManagement.scala

```diff
@@ -155,6 +156,59 @@ class HttpServiceIntegrationTestUserManagementNoAuth
       }
     } yield assertion
   }
+
+  import com.daml.http.json.JsonProtocol._
+  "requesting the user id should be possible via the user endpoint" in withHttpServiceAndClient(
+    participantAdminJwt
+  ) { (uri, _, _, ledgerClient, _) =>
```

Comment:
```suggestion
  "requesting the user id should be possible via the user endpoint" in withHttpServiceAndClient(
    participantAdminJwt
  ) { (uri, _, _, ledgerClient, _) =>
    import com.daml.http.json.JsonProtocol._
```

I think this is the scope you're looking for.